components/support/nimbus-fml/src/backends/swift/templates/FeatureTemplate.swift

```diff
@@ -0,0 +1,44 @@
+{%- import "macros.swift" as swift %}
+{%- let inner = self.inner() %}
+{%- let raw_name = inner.name() %}
+{% let class_name = inner.name()|class_name -%}
+
+{{ inner.doc()|comment("") }}
+public class {{class_name}} {
+    private var _variables: Variables
```

Comment:
This is smart here: this will make it easier later on when adding `Text` and `Image`.